### PR TITLE
MAGECLOUD-2447: Fix code style to fit Magento's

### PR DESCRIPTION
**Pull request by shiftedreality**

### CODE REVIEW

File: Console/Command/StorageExportCommand.php [hunk 1]
```diff
@@ -1,143 +1,156 @@
...
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        try {
-            $this->state->setAreaCode('adminhtml');
-        } catch (\Magento\Framework\Exception\LocalizedException $e) {
```
Comment (michaelcasey316) [inline Console/Command/StorageExportCommand.php:51]: Did you remove this in the new version? Why so?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /.idea/
+composer.lock
```
Comment (bbatsche) [inline .gitignore:2]: Should also include `/vendor/`

File: Console/Command/ConfigListCommand.php [hunk 3]
```diff
@@ -2,44 +2,76 @@
...
+     * @inheritdoc
+     */
     protected function configure()
     {
         $this->setName('s3:config:list');
         $this->setDescription('Lists whatever credentials for S3 you have provided for Magento.');
     }
 
+    /**
+     * {@inheritdoc}
```
Comment (bbatsche) [inline Console/Command/ConfigListCommand.php:47]: I don't think we should use `@inheritdoc` here. The parent documentation is from Symfony about why the parent method is not abstract; it's not relevant to this implementation.

File: Console/Command/ConfigSetCommand.php [hunk 4]
```diff
@@ -2,77 +2,111 @@
...
 
-        $output->writeln('<info>You have successfully updated your S3 credentials.</info>');
+            $output->writeln('<info>You have successfully updated your S3 credentials.</info>');
+        });
     }
 
+    /**
+     * @return array
+     */
     public function getOptions()
```
Comment (bbatsche) [inline Console/Command/ConfigSetCommand.php:99]: Should we be type hinting return types as well?

File: Console/Command/StorageEnableCommand.php [hunk 5]
```diff
@@ -44,92 +62,95 @@ protected function configure()
...
-                    'key' => $this->helper->getAccessKey(),
-                    'secret' => $this->helper->getSecretKey()
-                ]
-            ];
-
-            if ($this->helper->getEndpointEnabled()) {
-                if ($this->helper->getEndpoint()) {
-                    $options['endpoint'] = $this->helper->getEndpoint();
-                }
+                return;
```
Comment (bbatsche) [inline Console/Command/StorageEnableCommand.php]: These error conditions should return some non-zero value

File: Console/Command/StorageEnableCommand.php [hunk 6]
```diff
@@ -44,92 +62,95 @@ protected function configure()
...
+     * @throws \Exception
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        try {
-            $this->state->setAreaCode('adminhtml');
-        } catch (\Magento\Framework\Exception\LocalizedException $e) {
-            // intentionally left empty
-        }
+        $this->state->emulateAreaCode(Area::AREA_ADMINHTML, function () use ($output) {
```
Comment (bbatsche) [inline Console/Command/StorageEnableCommand.php]: Should return the value from this call.

File: composer.json [hunk 7]
```diff
@@ -18,6 +18,7 @@
     }
   },
   "require": {
-    "aws/aws-sdk-php": "3.x"
+    "aws/aws-sdk-php": "3.x",
+    "symfony/console": "^2.6||^4.0"
```
Comment (bbatsche) [inline composer.json]: Seems like there's a lot of Magento framework & module libraries that are used, should we not list them as well here?

**Review by bbatsche — CHANGES_REQUESTED** [review]
Mostly just questions for clarification

File: .gitignore (hunk 2 above)
Comment (shiftedreality) [reply .gitignore:2]: Added

File: Console/Command/ConfigSetCommand.php (hunk 4 above)
Comment (shiftedreality) [reply Console/Command/ConfigSetCommand.php:99]: It's currently compatible with 5.6, so no type hinting

File: Console/Command/ConfigListCommand.php (hunk 3 above)
Comment (shiftedreality) [reply Console/Command/ConfigListCommand.php:47]: It has return code descriptions and all `execute` commands are pretty same

File: Console/Command/StorageEnableCommand.php (hunk 6 above)
Comment (shiftedreality) [reply Console/Command/StorageEnableCommand.php]: Added

File: composer.json (hunk 7 above)
Comment (shiftedreality) [reply composer.json]: Added

**Review by bbatsche — APPROVED** [review]

**Review by michaelcasey316 — CHANGES_REQUESTED** [review]

File: Console/Command/StorageExportCommand.php (hunk 1 above)
Comment (shiftedreality) [reply Console/Command/StorageExportCommand.php:51]: This is done via `emulateAreaCode`, but in the correct way with restoring the previous state

**Review by michaelcasey316 — APPROVED** [review]

**Pull request merged**